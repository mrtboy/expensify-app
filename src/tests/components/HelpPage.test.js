import React from 'react';
import HelpPage from '../../components/HelpPage';
import { shallow } from 'enzyme';

test("should render HelpPage", () =>{
  const wrapper = shallow(<HelpPage />);
  expect(wrapper).toMatchSnapshot();
}); 