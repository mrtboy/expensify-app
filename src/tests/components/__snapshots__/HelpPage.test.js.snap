// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render HelpPage 1`] = `
<div>
  HelpPage
</div>
`;
